package.json

```diff
@@ -18,9 +18,10 @@
   },
   "scripts": {
     "clean": "rm -r dist",
-    "lint": "eslint . && prettier . --check && npm run rule-syntax-check",
+    "lint": "eslint . && prettier . --check && npm run validate-rules",
     "bundle": "./build.sh",
-    "rule-syntax-check": "node scripts/validate-json-rules.js",
+    "generate-rule-schema": "ts-node scripts/generate-rule-schema.ts",
+    "validate-rules": "ts-node scripts/validate-rules.ts",
```

Comment:
`ts-node` is a new dependency that we'd need to consider carefully. For now, sticking with plain JS for CLI scripts and using JSDoc comments for typing would be more consistent with other DDG repos.

Reply:
Is there a particular concern you have with `ts-node`? `get-text-for-xpath` (https://github.com/duckduckgo/autoconsent/pull/386) is an existing script that uses it. I think TypeScript is nice for scripts as it gives you a little bit of extra confidence in an environment that doesn’t usually have unit tests. But it's not a strong opinion, happy to defer to you on this.